app/src/main/java/com/hadiyarajesh/notex/MyApplication.kt

```diff
@@ -1,7 +1,36 @@
 package com.hadiyarajesh.notex
 
 import android.app.Application
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.os.Build
+import com.hadiyarajesh.notex.utility.Constants.CHANNEL_DESCRIPTION
+import com.hadiyarajesh.notex.utility.Constants.CHANNEL_ID
+import com.hadiyarajesh.notex.utility.Constants.CHANNEL_NAME
 import dagger.hilt.android.HiltAndroidApp
 
 @HiltAndroidApp
-class MyApplication: Application()
+class MyApplication: Application(){
+
+
+    override fun onCreate() {
+        super.onCreate()
+        createNotificationChannels();
+    }
+    private fun createNotificationChannels() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
what's for the case when Build.VERSION_CODES below Oreo.

Reply:
Starting with Oreo, you need a notification channel if you want to post notification.